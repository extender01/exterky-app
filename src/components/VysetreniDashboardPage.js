import React from "react";
import SeznamVysetreni from "./SeznamVysetreni";
import VysetreniFiltrovaciInput from "./VysetreniFiltrovaciInput";




const VysetreniDashboardPage = () => (
    <div>
        <h4>Tohle je dashboard</h4>
        <VysetreniFiltrovaciInput />
        <SeznamVysetreni />
    </div>
);

export default VysetreniDashboardPage;