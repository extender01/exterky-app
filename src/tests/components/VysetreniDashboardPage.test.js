import React from "react";
import { shallow } from "enzyme";
import VysetreniDashboardPage from "../../components/VysetreniDashboardPage";


test("should render expense dashboard page", () => {
    const wrapper = shallow(<VysetreniDashboardPage />);
    expect(wrapper).toMatchSnapshot();

});