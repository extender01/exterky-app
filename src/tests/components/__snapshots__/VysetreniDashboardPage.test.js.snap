// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense dashboard page 1`] = `
<div>
  <h4>
    Tohle je dashboard
  </h4>
  <Connect(VysetreniFiltrovaciInput) />
  <Connect(SeznamVysetreni) />
</div>
`;
